components/sup/doc/api.raml

```diff
@@ -0,0 +1,75 @@
+#%RAML 0.8
+---
+title: Habitat Supervisor API
+
+baseUri: https://{rootUri}
```

Comment:
This should be http, not https, unless we're auto-generating certificates. That's probably a neat idea, actually, but it wouldn't prove a ton security wise.